src/routes/tutorials/hello-world.svelte

```diff
@@ -0,0 +1,247 @@
+<script>
+import SideNav from "$lib/components/sections/layout/SideNav.svelte";
```

Comment:
Why are individual pages hardcoded and not rendered from github readme via svelte mdx? https://madewithsvelte.com/mdsvex

Reply:
Hey Andrea. This is my first Svelte project so went as vanilla as possible. This was done to get the pages working so that we can show the progress in Town Hall. 
please drop me a list of changes and I'll hop on them